app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md

```diff
@@ -23,15 +23,29 @@ SKE courses can help when you have:
 - a different but related degree to the subject you want to teach
 - an A level but not a degree in the subject
 - an unrelated degree but relevant professional experience
-- a languages degree but need to improve your knowledge of a language you’ll teach
+- studied for a languages degree, but need to develop a second language to an acceptable level for teaching in schools 
 - not used your subject knowledge for a while
 
-When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge.
+When you apply for teacher training, the school or university will help you establish whether you need to improve your subject knowledge. They may decide to make completing a SKE course a condition of your offer to train with them.
 
-Completing a course may be a condition of your offer to train.
+SKE courses are currently offered in 9 secondary subjects:  
 
-They’re typically offered for subjects where there’s a particular demand for teachers.
+- biology 
+- chemistry 
+- computing 
+- design and technology 
+- English
+- languages 
+- mathematics
+- physics 
+- religious education
 
-Courses are available across England and are often online. You can do them before or alongside your teacher training. You may also be able to study part-time.
+You can also do a SKE course in primary mathematics.
 
-[Find out if there's a subject knowledge enhancement course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory).
+SKE course lengths can range from 8 to 28 weeks. They're usually online, but some can be classroom based and some can be a mixture of both. You can choose the provider you want to complete your SKE course with.
+
+Courses are available across England. Most candidates complete a SKE course before starting their teacher training year, but it’s sometimes possible to complete it alongside your initial teacher training (ITT) course. You may be able to complete the SKE course on a part-time basis.
+
+You may also be eligible for a bursary of £175 per week if you train to teach in the 2022/23 academic year. This will go towards any extra costs you may have during the SKE course. Contact your ITT and SKE providers for more information.  
```

Comment:
This feels a little odd - the bursary is for people doing ITT this year but the people reading this page will be looking at ITT in future years? The sentence makes it sound like the bursary is only for people this year; whilst we don't know whether it will be available next year, it might be worth rewording a little? But this can be done in retrospect post go-live